src/javascripts/components/cookie-banner.mjs

```diff
@@ -12,12 +12,12 @@ const cookieConfirmationRejectSelector = '.js-cookie-banner-confirmation-reject'
 /**
  * Website cookie banner
  */
-class CookieBanner extends Component {
+class CookieBanner extends ConfigurableComponent {
   /**
    * Check support of CookieBanner
    */
   static checkSupport() {
-    Component.checkSupport()
+    ConfigurableComponent.checkSupport()
```

Comment:
**suggestion** I think [`super`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super#super-calling_static_methods) should work here to avoid further updates 😁  

```suggestion
    super.checkSupport()
```